outputs.tf

```diff
@@ -44,3 +44,36 @@ output "enterprise_accounts" {
     }]
   ])
 }
+
+output "enterprise_accounts_iam_response" {
```

Comment:
Could you add outputs to the examples which show how a consumer could extract details from this output. Because the output has been marked as sensitive, terraform won't print it to the console, so it might be handy to add some outputs in the examples which show how a consumer could extract the apikey value maybe